### DISCUSSION TITLE
Improved search

### DISCUSSION THREAD

**Discussion by AlUlkesh** [AlUlkesh/stable-diffusion-webui-images-browser#255] [Announcements]
I have improved the search function. You can now use a simple AND/OR/NOT logic. For example:\
`robot and organic`\
`robot or organic`\
`robot and not organic`

Please be aware that due to performance considerations complex searches and parentheses are not supported.

![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/99896447/48c4dd93-20b7-4f6e-8a86-a2cf4bd8cb5a)

**Comment by bluntstone** [comment]
thank you for all your hard work on this. Really does make a difference.

**Reply by AlUlkesh** [reply]
Thank you 😄
